.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+| Question      | Answer
+| ------------- | ---
+| Tickets       | <!-- URLs to GitHub or JIRA issue(s) (or N/A) -->
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Tests pass?   | yes/no
+| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)
```

Comment:
So what happens if someone posts PR with License different than what we have in LICENSE file? :)

Reply:
You know it's just a template, to be edited right? :P

Not to mention, he can't: https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.html#GPLModuleLicense

Reply:
My point is, if this cannot be changed, there's no point of exposing that information. But if you prefer to hint license, I won't mind. We can adapt later if this turns to be what I expect it to be :)

Reply:
>  if this cannot be changed

It can. But it have to be GPL-2.0 compatible. (ie. The Unlicense, Public Domain or MPL 2.0). I've added comment:

`<!-- Keep in mind: Your contribution has to be compatible with GPL-2.0 as well: https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.html#GPLModuleLicense -->`

To make it clear. :)